tests/baselines/reference/tsbuild/emptyFiles/initial-build/does-not-have-empty-files-diagnostic-when-files-is-empty-and-references-are-provided.js

```diff
@@ -0,0 +1,6 @@
+//// [/lib/initial-buildOutput.txt]
+/lib/tsc --b /src/no-references
+src/no-references/tsconfig.json(3,14): error TS18002: The 'files' list in config file '/src/no-references/tsconfig.json' is empty.

```

Comment:
there is an error here. does that agree with the intent of the test "does not have empty files diagnostic"?

Reply:
Oops switched the names of hte scenarios.. Thanks for catching it.. Fixed it